docs/application/native/guides/app-management/data-control.md

```diff
@@ -1030,7 +1030,7 @@ The data model must be opened to the public to help other applications to use th
 - Trusted
   - You can allow access from other applications signed with the same certificate by setting the trusted status for the data control.
 - Privileges
-  - Your provider application can restrict access to applications having certain predefined privileges.
```

Comment:
It is possible to use app-defined privileges also

Reply:
Thank you for the explanation.

I accept the comment.